ale/drivers/lro_drivers.py

```diff
@@ -505,3 +505,59 @@ def spacecraft_direction(self):
         rotation = frame_chain.compute_rotation(1, lro_bus_id)
         rotated_velocity = spice.mxv(rotation._rots.as_dcm()[0], velocity)
         return rotated_velocity[0]
+
+
+class LroMiniRfIsisLabelNaifSpiceDriver(Radar, NaifSpice, IsisLabel, Driver):
```

Comment:
This should inherit from IsisSpice because we're going to allow ISIS to spiceinit right now.